pkg/koordlet/runtimehooks/reconciler/reconciler.go

```diff
@@ -56,21 +76,26 @@ type cgroupReconciler struct {
 
 type reconcileFunc func(protocol.HooksProtocol) error
 
-func RegisterCgroupReconciler(level ReconcilerLevel, cgroupFile system.CgroupFile,
+func RegisterCgroupReconciler(level ReconcilerLevel, qosClasses []apiext.QoSClass, cgroupFile system.CgroupFile,
```

Comment:
how about define a new struct to express the “Filter Condition”, so that we dont need to change api if new condition added

Reply:
done. add a Filter interface